### ISSUE TITLE
installation stops with WARNING: 'aclocal-1.15' is missing on your system

### ISSUE THREAD

**Issue by rsajan** [Jasig/mod_auth_cas#97]
I ran into a small issue during installation and I'm listing here FYI and for anyone that might be having problems with this. 

After I start the installation: 

`$ ./configure && make && sudo make install`

I receive this during the installation process:  

```
/home/myuser/source/mod_auth_cas/mod_auth_cas/missing: line 81: aclocal-1.15: command not found
WARNING: 'aclocal-1.15' is missing on your system.
         You should only need it if you modified 'acinclude.m4' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'aclocal' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
         It also requires GNU Autoconf, GNU m4 and Perl in order to run:
         <http://www.gnu.org/software/autoconf>
         <http://www.gnu.org/software/m4/>
         <http://www.perl.org/>
make: *** [aclocal.m4] Error 127
```

Then I need to run autoconf: 

$ autoreconf -f -i

Then run the install one more time: 

$ ./configure && make && sudo make install

And now it installed.

**Comment by dhawes** [comment]
See #91.

The short version is that cloning from git does not preserve create timestamps, so a rebuild is triggered.

Running autoconf like you have done will fix this issue. I have also found 'touch *' to get around the issue as well.

**Comment by dhawes** [comment]
A note was added to the README included in the v1.1 release:

https://github.com/Jasig/mod_auth_cas/blob/master/README#L111

**Comment by MounicaChV** [comment]
> I ran into a small issue during installation and I'm listing here FYI and for anyone that might be having problems with this.
> 
> After I start the installation:
> 
> `$ ./configure && make && sudo make install`
> 
> I receive this during the installation process:
> 
> ```
> /home/myuser/source/mod_auth_cas/mod_auth_cas/missing: line 81: aclocal-1.15: command not found
> WARNING: 'aclocal-1.15' is missing on your system.
>          You should only need it if you modified 'acinclude.m4' or
>          'configure.ac' or m4 files included by 'configure.ac'.
>          The 'aclocal' program is part of the GNU Automake package:
>          <http://www.gnu.org/software/automake>
>          It also requires GNU Autoconf, GNU m4 and Perl in order to run:
>          <http://www.gnu.org/software/autoconf>
>          <http://www.gnu.org/software/m4/>
>          <http://www.perl.org/>
> make: *** [aclocal.m4] Error 127
> ```
> 
> Then I need to run autoconf:
> 
> $ autoreconf -f -i
> 
> Then run the install one more time:
> 
> $ ./configure && make && sudo make install
> 
> And now it installed.

Thank a lot!!!!! My error resolved after following your steps
